test/JuMP_sparse_linear_mpc.jl

```diff
@@ -0,0 +1,76 @@
+using Ipopt, JuMP, Random, LinearAlgebra
```

Comment:
We need to specify the dependency for test in `Project.toml`. Check https://github.com/MadNLP/MadNLP.jl/blob/master/Project.toml#L27-L34